src/main/java/seedu/address/model/Entity/Participant.java

```diff
@@ -0,0 +1,141 @@
+package seedu.address.model.Entity;
+
+import java.util.HashMap;
+import java.util.Objects;
+
+public class Participant extends Entity {
+    private Email email;
+    private Phone phone;
+    private Team team;
+
+    /**
+     * Constructor without team.
+     *
+     * @param name
+     * @param email
+     * @param phone
+     * @param id
+     */
+    public Participant(Name name, Email email, Phone phone, Id id) {
+        super(id, name);
+        this.email = email;
+        this.phone = phone;
+        // TODO: Make this case impossible? because participants cannot exist w/o team
+        this.team = null;
+    }
+
+    public Participant(Name name, Email email, Phone phone, Id id, Team team) {
```

Comment:
Hey. As discussed during our last meeting, Participant will no longer have a reference to Team, so this constructor can be removed.

Reply:
@brianyenna Removed this in #13.